packages/react-core/src/next/components/Select/Select.tsx

```diff
@@ -30,6 +30,8 @@ export interface SelectProps extends MenuProps, OUIAProps {
   zIndex?: number;
   /** @beta Determines the accessible role of the select. For a checkbox select pass in "menu". */
   role?: string;
+  /** Additional properties to pass to the popper */
+  popperProps?: PopperProps;
```

Comment:
So required props are not needed:
```suggestion
  popperProps?: Partial<PopperProps>;
```